openassessment/templates/openassessmentblock/edit/oa_edit.html

```diff
@@ -139,9 +139,9 @@ <h2 class="openassessment_alert_title">{% trans "Rubric Change Impacts Settings
 
             <p class="openassessment_description" id="openassessment_step_select_description">
                 {% if 'example_based_assessment' in editor_assessments_order %}
-                    {% trans "In this assignment, you can include steps for student training, peer assessment, self assessment, and example based assessment. Select the steps that you want below, and then drag them into the order that you want. If you include an example based assessment step, it must precede all other steps. If you include a student training training step, it must precede peer and self assessment steps." %}
+                    {% trans "In this assignment, you can include steps for learner training, peer assessment, self assessment, and example based assessment. Select the steps that you want below, and then drag them into the order that you want. If you include an example based assessment step, it must precede all other steps. If you include a learner training training step, it must precede peer and self assessment steps." %}
```

Comment:
In "If you include a learner training training step", the word "training" is repeated; delete one of the occurrences.